languages/en.php

```diff
@@ -41,6 +41,7 @@
 	'actionunauthorized' => 'You are unauthorized to perform this action',
 	
 	'ajax:error' => 'Unexpected error while performing an AJAX call. Maybe the connection to the server is lost.',
+	'ajax:not_is_xhr' => 'AJAX: It is not allowed to access these AJAX views directly.',
```

Comment:
The wording for this message seems clunky.

"You cannot access AJAX views directly"
"Accessing AJAX views directly is not allowed"
"AJAX views are not directly accessible"

?